src/main/javascript/service/cat-validation-service.js

```diff
@@ -32,6 +32,30 @@ function CatValidationService($log,
     var that = this;
 
     /**
+     * Adds a field error to the context
+     * @param fieldName name of the faulty field
+     * @param errorMessage associated error message to display
+     * @param context affected context
+     */
+    function appendFieldErrorToContext(fieldName, errorMessage, context) {
+        if (catMessagesConfig.knownFieldsActive === true) {
+            // If the error is for a known field, show the error at the field.
+            // If not, display it as a global error.
+            if (_.contains(context.knownFields, fieldName)) {
```

Comment:
I didn't know that you are currently in the process of making it compatible with lodash 4. 
Thanks for that advice and for taking care of it!